docs/messaging/messaging-components.md

```diff
@@ -221,35 +341,45 @@ app.MapPost("/notify", static async (ServiceBusClient client, string message) =>
 
     Console.WriteLine($"A message has been published to the topic.");
 });
+
+app.MapDefaultEndpoints();
+
+app.Run();
 ```
 
 ## Add the .NET Aspire component to the Worker Service
 
-Add the [.NET Aspire Azure Service Bus](azure-service-bus-component.md) component to your `AspireMessaging.WorkerService` app:
+Add the [.NET Aspire Azure Service Bus](azure-service-bus-component.md) component to your **AspireSample.WorkerService** project. Follow the same steps as you did before when you added the [Aspire.Azure.Messaging.ServiceBus](https://nuget.org/packages/Aspire.Azure.Messaging.ServiceBus) NuGet package to the **AspireSample.ApiService** project. Once it's been added, you can configure the worker service to process messages from the Service Bus topic.
```

Comment:
What's the Service Bus "topic"?

Reply:
Service Bus has the concept of "topics", in addition to "queues" and "subs": https://learn.microsoft.com/azure/service-bus-messaging/service-bus-queues-topics-subscriptions. Folks who are familiar with Service Bus will know what that is. Do you think it needs more info?